### ISSUE TITLE
Missing comma in docs

### ISSUE THREAD

**Issue by matankb** [Dogfalo/materialize#541]
In the [dropdown docs](http://materializecss.com/dropdown.html), there is a missing comma in the jQuery Plugin Initialization code. Currently, it says
```
$('.dropdown-button').dropdown({
      inDuration: 300,
      outDuration: 225,
      constrain_width: false, // Does not change width of dropdown to that of the activator
      hover: false // Activate on click
      alignment: 'left', // Aligns dropdown to left or right edge (works with constrain_width)
      gutter: 0, // Spacing from edge
    }
  );
```
On line five, there should be a comma directly after `hover: false`.

**Comment by drKraken** [comment]
gutter: 0, // Spacing from edge
And should not be here.

**Comment by acburst** [comment]
Ahh thanks for catching that.

Fixed in acfb5d35bb1b4c929e099ba1ca1069556294cd4e
